Source/Fuse.Common/Internal/FontListParser.java

```diff
@@ -224,6 +219,29 @@ private static Family readFamily(XmlPullParser parser)
         return new Family(names, fonts, lang, variant);
     }
 
+    /** Matches leading and trailing XML whitespace. */
+    private static final Pattern FILENAME_WHITESPACE_PATTERN =
+            Pattern.compile("^[ \\n\\r\\t]+|[ \\n\\r\\t]+$");
+
+    private static Font readFont(XmlPullParser parser)
+            throws XmlPullParserException, IOException {
+        String indexStr = parser.getAttributeValue(null, "index");
+        int index = indexStr == null ? 0 : Integer.parseInt(indexStr);
+        String weightStr = parser.getAttributeValue(null, "weight");
+        int weight = weightStr == null ? 400 : Integer.parseInt(weightStr);
```

Comment:
Could this be NormalWeight? or is it just coincidentally the same value?

Reply:
Dunno. I just took this verbatim from [upstream](https://raw.githubusercontent.com/aosp-mirror/platform_frameworks_base/master/graphics/java/android/graphics/FontListParser.java), and I'd prefer to keep it as similar as that to avoid needless divergence...

Reply:
nice then time to approve :)